idseq_dag/steps/blast_contigs.py

```diff
@@ -99,10 +99,10 @@ def summary(self):
             r["length"] = 1
         r["pident"] = sum(hsp["pident"] * hsp["length"] for hsp in self.optimal_cover) / r["length"]
         r["bitscore"] = sum(hsp["bitscore"] for hsp in self.optimal_cover)
-        r["qstart"] = min(hsp["qstart"] for hsp in self.optimal_cover)
-        r["qend"] = max(hsp["qend"] for hsp in self.optimal_cover)
-        r["sstart"] = min(hsp["sstart"] for hsp in self.optimal_cover)
-        r["send"] = max(hsp["send"] for hsp in self.optimal_cover)
+        r["qstart"] = min(min(hsp["qstart"], hsp["qend"]) for hsp in self.optimal_cover)
```

Comment:
please add a code comment why `qstart` and `qend` may need to be flipped. it's not obvious, at least to me 🙂

Reply:
great suggestion, will add this brief note
```
# depending on strand, qstart and qend may be reversed
```